libs/components/phone-field/src/lib/modules/phone-field/phone-field-input.directive.ts

```diff
@@ -64,14 +64,16 @@ export class SkyPhoneFieldInputDirective
    * @default false
    */
   @Input()
-  public set disabled(value: boolean) {
-    this.phoneFieldComponent.countrySelectDisabled = value;
-    this.adapterService.setElementDisabledState(this.elRef, value);
-    this._disabled = value;
+  public set disabled(value: boolean | undefined) {
+    if (this.#phoneFieldComponent && value) {
```

Comment:
I looks like we're only updating the other disabled properties if the value is truthy, but it should allow setting them to `false`, yes?

Reply:
Correct. Good catch.